### DISCUSSION TITLE
Deadlock when 2 features try to use the same directory

### DISCUSSION THREAD

**Discussion by Hyphrio** [community/community#49884] [Codespaces]
### Select Topic Area

Bug

### Body

Creating a codespace with both Homebrew and Homebrew-Install causes the codespace to deadlock when rebuilt.

To replicate:
Create a codespace
Add both Homebrew and Homebrew install
Rebuild container:

Expected behaviour:
Either the codespace start successfully or returns you to recovery

I think you could maybe add an option to go to recovery or you could detect deadlocks?

**Comment by jungaretti** [comment]
Could you please share the full name of the two features that you're working with? It sounds like they may need to use the `installsAfter` property: https://containers.dev/implementors/features/

**Reply by Hyphrio** [reply]
Sorry for the late answer but those are Homebrew and Homebrew Install.
